tests/unit/inputs/test_surrogate_th.xml

```diff
@@ -1,13 +1,11 @@
 <?xml version="1.0"?>
 <stream>
-  <driver_transport>openmc</driver_transport>
-  <driver_heatfluids>surrogate</driver_heatfluids>
-  <communication>overlapping</communication>
-  <power>820.0</power>
-  <pressure_bc>12.7553</pressure_bc>
-  <max_timesteps>1</max_timesteps>
-  <max_picard_iter>4</max_picard_iter>
-  <heat_surrogate>
+  <neutronics>
+    <driver>openmc</driver>
+  </neutronics>
+  <heat_fluids>
+    <driver>surrogate</driver>
+    <pressure_bc>101325.0</pressure_bc>
```

Comment:
This value of `<pressure_bc>` is the outlet temperature used in `tests/unit/test_surrogate_th.cpp`.  Rather than being set in test driver, I'm setting it here.